tests/test_worker.py

```diff
@@ -634,6 +634,15 @@ def test_timeouts(self):
         res.refresh()
         self.assertIn('JobTimeoutException', as_text(res.exc_info))
 
+    def test_dequeue_job_and_maintain_ttl_non_blocking(self):
+        """Not passing a timeout should return immediately with None as a result"""
+        q = Queue()
+        w = Worker([q])
+
+        # Put it on the queue with a timeout value
+        res = w.dequeue_job_and_maintain_ttl(None)
+        assert res is None
```

Comment:
Mind changing this to `self.assertIsNone(w.dequeue_job_and_maintain_ttl(None))`?